api/age-group/models/age-group.js

```diff
@@ -8,6 +8,59 @@ const { createLifecycleHooks } = require("../../../utils/algolia");
 
 const contentType = "age-group";
 
+const hooks = createLifecycleHooks(contentType);
+
+const updateActivityGroupActivities = async (group, ageGroupId) => {
+  const activityGroup = await strapi
+    .query("activity-group")
+    .findOne({ id: group.id });
+
+  if (activityGroup.activity_groups?.length) {
```

Comment:
This now runs even if activity_groups is an empty array?  Dont know if this is really a problem tho, since mapping is done with `for const g of array`, syntax.
    
Can this in any case pass undefined to updateActivityGroupActivities function?

Reply:
No it doesn't. This runs only if `activityGroup.activity_groups` is defined and has items in it.

In the case of an empty array, the length is `0`, and it is a falsy value interpreted as `false` by the if-block.